### ISSUE TITLE
Add support for new newznab bookstrap theme

### ISSUE THREAD

**Issue by lilHermit** [gboudreau/sabconnectplusplus#141]
Please add support for the new newznab theme/template called bookstrap. Please see screengrab
![screenshot from 2016-05-03 14-47-28](https://cloud.githubusercontent.com/assets/299424/14985210/5f80e382-113e-11e6-8b88-3a4df14cfbac.png)

**Comment by ppslim** [comment]
Do you have access to such a service and willing to provide some technical details to assist?

Without access to a page rendered in that theme, there is little to see on why the current selectors do not work, or what needs adding / changed to get it running.

**Comment by lilHermit** [comment]
Yeah I can give you access to an install how do you suggest I pass over the details?

**Comment by lilHermit** [comment]
*bump* ;-)

**Comment by ppslim** [comment]
Sorry, I had not seen a way for you to send it.

I note the email address may now be invalid, but I have emailed it regardless. It may be best to try and catch me before the weekend as time is not my friend recently.

**Comment by lilHermit** [comment]
Ok try email me here lilhermit@forward.cat and we'll get things rolling. I can certainly set you up asap

**Comment by ppslim** [comment]
Having had a quick scan, there are a number of factors.

Detecting and inserting looks does not appear to be too difficult, however...

The template purposefully comments out the RSSTOKEN input field and means the API key cannot be extracted from the DOM easily.

The API for getting the NZB file itself looks to be very different, which may mean a different submission mechanism.

Overall, this may be difficult and easy in one. I should get around to looking in more detail this weekend.

**Comment by lilHermit** [comment]
Ok let me know if there anything you need. In the meantime I'll do a source update to see if anything has changed with the theme

**Comment by ppslim** [comment]
I have done a first pass at the selectors required for this.

The page structure has changed a lot, but that isn't unexpected given this is a Bootstrap page.

The List & Cover views are covered in this, however, they will now download to sab yet, just render links.

You would need to install a local extension to test this after downloading from my iss141 branch ppslim/sabconnectplusplus@218d9b8

**Comment by lilHermit** [comment]
Looks good I see the rendering of the buttons!
